### PR TITLE
Dh ak footer

### PR DESCRIPTION
**Pull request by AdamKnowles**
Please look at the footer, make sure all links work please.

### CODE REVIEW

File: main.css [hunk 1]
```diff
@@ -29,3 +29,30 @@ main article img {
 /* Joy's CSS Changes From Here */
 
 /* Joy's CSS Changes To Here */
+
+.all-footer {
```
Comment (AdamKnowles) [inline main.css:59]: Looking good

**Comment by AdamKnowles** [comment]
👍 Nice!

**Comment by joekennerly** [comment]
this is great but I want to merge first

**Comment by joekennerly** [comment]
good job managing that stressful merge. thanks for everything you do!
btw :+1:

**Pull request merged**